AutoscoperM/AutoscoperM.py

```diff
@@ -900,10 +904,10 @@ def progressCallback(x):
             filename = os.path.join(outputDir, volumeSubDir, segmentName + ".tif")
             IO.castVolumeForTIFF(segmentVolume)
             IO.writeVolume(segmentVolume, filename)
-            spacing = segmentVolume.GetSpacing()
```

Comment:
```suggestion
            spacing = segmentVolume.GetSpacing()
```

Unless changing the order is needed, I suggest to keep it as such.